src/syscheckd/src/recovery/recovery.cpp

```diff
@@ -0,0 +1,154 @@
+#include <cstdint>
+#include <vector>
+#include <string>
+#include <hashHelper.h>
+
+extern "C" {
+    #include "debug_op.h" // Use the same C-style logging as the rest of FIM's code.
+}
+
+#include "recovery.h"
+#include "agent_sync_protocol_c_interface.h"
+#include "agent_sync_protocol_c_wrapper.hpp"
+#include "db.hpp"
+#include "fimCommonDefs.h"
+#include "stringHelper.h"
+#include "ipersistent_queue.hpp"
+#include "syscheck.h"
+#include <chrono>
+
+/**
+ * @brief Get the current timestamp in UNIX format
+ */
+int64_t getUnixTimeSeconds() {
```

Comment:
There is already a function `Utils::getSecondsFromEpoch()` (defined in `shared_modules/utils/timeHelper.h`) that performs exactly the same operation.